tests/jobQueue/v1/getBulkUpsertStatus.js

```diff
@@ -171,13 +173,13 @@ describe('tests/jobQueue/v1/getBulkUpsertStatus.js, ' +
     });
   });
 
-  it('Even when hundreds of upserts fail, the valid upsert should ' +
+  it('Even when lots of upserts fail, the valid upsert should ' +
```

Comment:
smaller number of samples could mean smaller setTimeout?
(500ms times multiple tests is expensive)